pinc/LPage.inc

```diff
@@ -137,10 +135,29 @@ function get_available_page($projectid, $proj_state, $pguser, &$err)
     }
 
     $round = get_Round_for_project_state($proj_state);
-    $err = can_user_get_pages_in_project($pguser, $project, $round);
-    if ($err) {
+
+    try {
+        $page_array = get_available_proof_page_array($project, $round, $pguser);
```

Comment:
Just use `[$imagefile, $state] = ` here instead of using a temp var as that would be clearer.